packages/cloud-cognitive/src/components/PageHeader/PageHeader.test.js

```diff
@@ -595,13 +607,14 @@ describe('PageHeader', () => {
     jest.spyOn(console, 'error').mockRestore();
   });
 
-  test('Works, for now, with deprecated props', () => {
+  test.only('Works, for now, with deprecated props', () => {
```

Comment:
We shouldn't check in a `test.only`

```suggestion
  test('Works, for now, with deprecated props', () => {
```

Reply:
Already resolved locally